TACC/Core-Portal
# task/FP-1166: Handle users who have no allocations

## Overview: ##
Users without allocations were getting auto-passed in onboarding which made the next step, in which their keys are pushed,
impossible. In order to prevent this users with no allocations will be put into User Wait for this onboarding step. 

## Related Jira tickets: ##

* [FP-1166](https://jira.tacc.utexas.edu/browse/FP-1166)

## Summary of Changes: ##
Updated onboarding workflow related to allocations. If a user does not have any active or inactive allocations, rather
than skipping to complete, the step will be put into User Wait until the user can confirm with a PI. 

## Testing Steps: ##
1. Log in as user ctacc05 (contact me for password if needed)
2. Allocation step should display User Wait

## UI Photos:

![allocationsonboardingstep](https://user-images.githubusercontent.com/1160892/139305999-76b12155-2011-44b5-8ef0-fae5b1786ed3.png)

## Notes: ##
* The MFA step is passing for me but if it fails simply remove it from your _PORTAL_USER_ACCOUNT_SETUP_STEPS array in settings_custom.py
* I have no idea what the description wording should be so if you want it changed I need to know what it should be set to
